ScreenConfig.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+import "brewblox.proto";
+import "nanopb.proto";
+
+package blox.ScreenConfig;
```

Comment:
Maybe not use `blox` as top-level package.

Reply:
fixed in f18dc1a6cdfbbd4f489edf5f8b2de641368e5911